### PR TITLE
config: enalbe Synopsys Ethernet Controllers stmmac

### PR DESCRIPTION
**Pull request by binbinwu1**
Add configs to enalbe Synopsys Ethernet Controllers "stmmac".

Signed-off-by: Binbin Wu <binbin.wu@intel.com>

### CODE REVIEW

File: recipes-kernel/linux/files/sos_5.4.cfg [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-# CONFIG_REFCOUNT_FULL is not set 
+# CONFIG_REFCOUNT_FULL is not set
+CONFIG_NET_VENDOR_STMICRO=y
+CONFIG_STMMAC_ETH=m
+CONFIG_STMMAC_PCI=m
+CONFIG_STMMAC_HWTS=y
+CONFIG_STMMAC_SELFTESTS=y
+CONFIG_DWXPCS=m
+CONFIG_MARVELL_PHY=m
+CONFIG_MARVELL_10G_PHY=m
```
Comment (anujm1) [inline recipes-kernel/linux/files/sos_5.4.cfg:10]: The last three above look unrelated. Also I can't find CONFIG_DWXPCS in mainline. Is it in ACRN kernel?

stmmac can be enabled by adding KERNEL_FEATURES_append = " features/net/stmicro/stmmac.scc"

Comment (binbinwu1) [reply recipes-kernel/linux/files/sos_5.4.cfg:10]: The device ACRN needs to support is using marvell phy and DWPCS is the PHY converter driver for DW xPCS IP.

Comment (binbinwu1) [reply recipes-kernel/linux/files/sos_5.4.cfg:10]: CONFIG_DWXPCS could be found in https://github.com/intel/linux-intel-lts/blob/5.4/yocto/

File: recipes-kernel/linux/linux-intel-acrn_4.19.inc [hunk 2]
```diff
@@ -23,4 +23,5 @@ KERNEL_FEATURES_append = " features/netfilter/netfilter.scc \
                            features/security/security.scc  \
                            cfg/hv-guest.scc \
                            cfg/paravirt_kvm.scc \
+                           features/net/stmicro/stmmac.scc \
```
Comment (saininav) [inline recipes-kernel/linux/linux-intel-acrn_4.19.inc:26]: Can you please change features/net/stmicro/stmmac.scc -> features/net/stmicro/stmmac.cfg in your patch at all the places !

Comment (binbinwu1) [reply recipes-kernel/linux/linux-intel-acrn_4.19.inc:26]: OK

**Pull request merged**